src/EventExport/FileWriter/TabularDataFileWriter.php

```diff
@@ -77,141 +70,232 @@ protected function includedOrDefaultProperties($include)
+                'name' => 'locatie naam',
+                'include' => function ($event) {
+                    if (property_exists($event, 'location') && isset($event->location->name)) {
+                        return $event->location->name;
+                    }
+                },
+                'property' => 'location'
+            ],
+            'address' => [
+                'name' => 'adres',
+                'include' => function ($event) {
+                    if (property_exists($event, 'location') &&
+                        isset($event->location->address)
+                    ) {
+                        $address = [];
+                        if (isset($event->location->address->streetAddress)) {
+                            $address[] = $event->location->address->streetAddress;
+                        }
 
-                    return implode("\r\n", $address);
-                }
-            }, 'property' => 'location' ],
-            'image' => [ 'name' => 'afbeelding', 'include' => function ($event) {
-                return $event->image;
-            }, 'property' => 'image' ],
-            'sameAs' => [ 'name' => 'externe ids', 'include' => function ($event) {
-                if ($event->sameAs) {
-                    $ids = array();
-
-                    foreach ($event->sameAs as $externalId) {
-                        $ids[] = $externalId;
+                        $line2 = [];
+                        if (isset($event->location->address->postalCode)) {
+                            $line2[] = $event->location->address->postalCode;
+                        }
+
+                        if (isset($event->location->address->addressLocality)) {
+                            $line2[] = $event->location->address->addressLocality;
+                        }
+
+                        if (!empty($line2)) {
+                            $address[] = implode(' ', $line2);
+                        }
+
+                        if (isset($event->location->address->addressCountry)) {
+                            $address[] = $event->location->address->addressCountry;
+                        }
+
+                        return implode("\r\n", $address);
                     }
+                },
+                'property' => 'location'
+            ],
+            'image' => [
+                'name' => 'afbeelding',
+                'include' => function ($event) {
+                    return $event->image;
+                },
+                'property' => 'image'
+            ],
+            'sameAs' => [
+                'name' => 'externe ids',
+                'include' => function ($event) {
+                    if (property_exists($event, 'sameAs')) {
+                        $ids = array();
 
-                    return implode("\r\n", $ids);
-                }
+                        foreach ($event->sameAs as $externalId) {
+                            $ids[] = $externalId;
+                        }
```

Comment:
The column callbacks now check if the required properties are set. Besides that only formatting should be changed.